src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -56,12 +59,31 @@ def _get_missing_members(self, organization: Organization) -> QuerySet[CommitAut
             .order_by("-commit_count")
         )
 
-    # TODO(cathy): check domain
+    def _get_email_domain(self, email: str) -> str:
+        return Address(addr_spec=email).domain
```

Comment:
Honestly I should make this into a helper function somewhere, I think I've recommended using this import for email stuff a couple times now :thinking: